spiffworkflow-frontend/test/browser/process_models/test_can_allow_searching_for_model.py

```
@@ -0,0 +1,57 @@
+import re
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)